### ISSUE TITLE
"Error calculating normalized scale of scene" in animated model

### ISSUE THREAD

**Issue by martinell** [openframeworks/openFrameworks#5866]
I have an animated 3D model where the animation is a rotation transform of the whole model around its center. When loading the model, I got an error saying "Error calculating normalized scale of scene".

When computing the normalized scale of the model, if it has animations, it does it based on the animated positions and ignores the model's geometry:

https://github.com/openframeworks/openFrameworks/blob/6b3b320fd2e8f21cde059df7a31dd4a9cf830e1e/addons/ofxAssimpModelLoader/src/ofxAssimpModelLoader.cpp#L630-L657

This can produce many unexpected results depending on the animations.

What could be a better option to fix this?
* Use the geometry only to compute the scale
* Play the animations and compute the scale based on the absolute min and max through all the transformed positions of the geometry

**Comment by arturoc** [comment]
yes this should take into account the vertices not the positions. I changed this cause sometimes animations imply scale which changes the size of the scene. Can you send a pull request?

**Comment by martinell** [comment]
Which of the two options I proposed do you want me to PR?

My guess is that the second one is a bit overkill, just to double check.

**Comment by arturoc** [comment]
No the second one is correct, as i said the original intention of this was to solve a problem where a model can be scaled by a lot when animated which gives a wrong value for the normalization factor. Perhaps as a compromise we could just use the first frame of animation instead of going through all of them
